.github/workflows/ci.yaml

```diff
@@ -67,8 +67,12 @@ jobs:
         run: |
           mkdir -p /tmp/gradle_test_reports/core
           mkdir -p /tmp/gradle_test_reports/schema-loader
+          mkdir -p /tmp/gradle_test_reports/data-loader/core
```

Comment:
Added here and not in separate data loader ci file as it seems easier to manage in this file like this.